content/static-pages/what-is-ebpf/index.pt.md

```diff
@@ -0,0 +1,289 @@
+---
+title: eBPF Documentation
+ogTitle: What is eBPF? An Introduction and Deep Dive into the eBPF Technology
+ogDescription: A detailed step by step introduction to the eBPF technology with lots of references for further reading.
+ogKeywords: ebpf, bpf, xdp, introduction, tutorial, what is, deep dive, documentation
+---
+
+## What is eBPF?
+
+O eBPF é uma tecnologia revolucionária com origens no kernel Linux que pode executar programas em um contexto privilegiado, como o kernel do sistema operacional. Ele é usado para estender com segurança e eficiência as capacidades do kernel sem a necessidade de alterar o código-fonte do kernel ou carregar módulos do kernel.
```

Comment:
The original text has the following sentence: ".. that can run **sandboxed** programs in a privileged context..."

The translation has no references to the term *sandboxed*, so the idea behind the context is lost. But this word does not have a direct translation into Portuguese. Commonly, when there is no corresponding term in Portuguese, we either leave the term in the source language and use italic formatting (example: "que pode executar programas em _sandbox_ em um contexto privilegiado" ) or look for a word that can carry a similar meaning. I would like to know the review preference in this case.

Reply:
I would probably go with the former so _sandbox_

Reply:
```suggestion
O eBPF é uma tecnologia revolucionária com origens no kernel Linux que pode executar programas em _sandbox_ em um contexto privilegiado, como o kernel do sistema operacional. Ele é usado para estender com segurança e eficiência as capacidades do kernel sem a necessidade de alterar o código-fonte do kernel ou carregar módulos do kernel.
```